### PR TITLE
figureMS3.svg

### PR DESCRIPTION
**Pull request by mcreixell**
First row is unclustered, second row k-means, third row GMM (pomegranate, not our method with weight=0). This is with the portion of the data set that is complete, 1311 peptides. For the first model, I'm just showing the peptides for which the cluster coefficients are not 0. For the other two, I run the clustering with 15 clusters each.
 
image

### CODE REVIEW

File: figureMS3.svg [hunk 1]
```diff
@@ -0,0 +1,4888 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (aarmey) [inline figureMS3.svg:1]: Don't think you meant to commit this?

**Review by aarmey — COMMENTED** [review]
Think this just shows that it's really easy to distinguish tumor from NAT. In this case, the cluster-by-cluster statistical testing is going to be more informative.

File: figureMS3.svg (hunk 1 above)
Comment (mcreixell) [reply figureMS3.svg]: deleted

Comment (mcreixell) [reply figureMS3.svg]: Do you mean running the statistical testing with k-means and GMM clusters as well?

Comment (aarmey) [reply figureMS3.svg]: I don't think you need to make that comparison everywhere.

**Review by aarmey — APPROVED** [review]

**Pull request merged**